ludwig/features/feature_registries.py

```diff
@@ -46,6 +46,11 @@
 from ludwig.features.text_feature import TextFeatureMixin, TextInputFeature, TextOutputFeature
 from ludwig.features.timeseries_feature import TimeseriesFeatureMixin, TimeseriesInputFeature
 from ludwig.features.vector_feature import VectorFeatureMixin, VectorInputFeature, VectorOutputFeature
+
+if TYPE_CHECKING:
```

Comment:
Interesting. Didn't realize this was an option for bypassing what I presume are flake8 errors?

Reply:
Yup, it prevents circular import errors, which can happen more frequently when typing with custom modules